### ISSUE TITLE
Menu dropdown padding is applied after styles added via createStyles

### ISSUE THREAD

**Issue by TheRarita** [mantinedev/mantine#1852]
### What package has an issue

@mantine/core

### Describe the bug

When using `createStyles` to for Menu.Dropdown with `classNames` to overwrite padding of dropdown, the padding style is applied before the default padding, which means it is ignored.
``` js

const useStyles = createStyles((theme) => ({
  dropdown: {
    padding: 12,
    backgroundColor: '#ccc'
  }
}));

...
  <Menu classNames={{ dropdown: classes.dropdown }}>
...

```

Generated css

``` CSS

.mantine-xxxxxx {
    position: absolute;
    background-color: #fff;
    background: #fff;
    border: 1px solid #e9ecef;
    padding: 12px 16px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.05),rgba(0, 0, 0, 0.05) 0px 20px 25px -5px,rgba(0, 0, 0, 0.04) 0px 10px 10px -5px;
    border-radius: 4px;
    padding: 12px; /* - custom style */
    background-color: #ccc;
    padding: 4px; /* - default style overwriting custom style */
}

```

see: https://codesandbox.io/s/admiring-aryabhata-2rj1xz?file=/src/Test.tsx

### In which browser did the problem occur

Chrome

### If possible, please include a link to a codesandbox with the reproduced problem

 https://codesandbox.io/s/admiring-aryabhata-2rj1xz?file=/src/Test.tsx

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

**Comment by raymonddaikon** [comment]
I have this problem too, but it's an easy fix, in MenuDropdown.tsx, Popover.Dropdown has the prop ```p={4}```, going to use patch package in the meantime.

**Comment by rtivital** [comment]
Fixed in 5.0.2
